packages/clerk-js/src/core/resources/User.ts

```diff
@@ -224,6 +224,10 @@ export class User extends BaseResource implements UserResource {
     });
   };
 
+  delete = (): Promise<void> => {
+    return this._baseDelete({ path: '/me' });
```

Comment:
Just want to flag this, as it's a somewhat non-standard path. Once we fully roll this out in the API there's no going back without a major version boost.